hooks/pre-deploy

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -eu
+
+if want_feature "shield-backups"; then
+    if ! [ -x "$(command -v shield)" ]; then
+        echo 'error: shield is not installed.' >&2
+        exit 1
+    fi
+
+    SHIELD_ADMIN_USERNAME="$(lookup --merged meta.shield.admin_username)"
+    SHIELD_ADMIN_PASSWORD="$(lookup --merged meta.shield.admin_password)"
+
+    SHIELD_ADDRESS="$(lookup --merged meta.shield.address)"
+
+    export BLACKSMITH_SHIELD_USERNAME="$(lookup --merged params.shield.username)"
+    export BLACKSMITH_SHIELD_PASSWORD="$(lookup --merged params.shield.password)"
+    export BLACKSMITH_SHIELD_TENANT="$(lookup --merged params.shield.tenant)"
+
+    shield api $SHIELD_ADDRESS blacksmith-shield -k
+    shield login -c blacksmith-shield --username $SHIELD_ADMIN_USERNAME --password $SHIELD_ADMIN_PASSWORD
+
+    TEMP_DIR=$(mktemp -d)
+
+    spruce merge manifests/templates/shield-backups-import.yml > $TEMP_DIR/output.yml
+    shield import -c blacksmith-shield $TEMP_DIR/output.yml
```

Comment:
You might be able to prevent writing to disk at all by using the following method:

```
shield import -c blacksmith-shield <(spruce merge manifests/templates/shield-backups-import.yml)
```